components/api/src/main/java/com/hotels/styx/api/client/loadbalancing/spi/LoadBalancingStrategyFactory.java

```diff
@@ -26,12 +27,24 @@
  * @see Environment
  */
 public interface LoadBalancingStrategyFactory extends ServiceFactory<LoadBalancingStrategy> {
+
+    /**
+     * Creates a strategy.
+     *
+     * @param environment           Styx application environment
+     * @param strategyConfiguration configuration specific to load balancer
+     * @return a new load balancing strategy.
+     */
+    default LoadBalancingStrategy create(Environment environment, Configuration strategyConfiguration) {
```

Comment:
Please remove this. We only need `create(Environment, Configuration, ActivieOrigins)` variant.